src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -678,6 +679,12 @@ def aks_create(
     workload_runtime=None,
     enable_custom_ca_trust=False,
 ):
+    if enable_cluster_snapshot:
+        header = "AKSHTTPCustomFeatures=AddonManagerV2Preview,AKSHTTPCustomFeatures=CSIControllersV2Preview"
+        if aks_custom_headers is None:
```

Comment:
@FumingZhang any comment function can used to update the custom header?

Reply:
I think the customer header should be like:
"AKSHTTPCustomFeatures=AddonManagerV2Preview,CSIControllersV2Preview"

if no commen function to call, then there would be issues like this

Reply:
No, it is not. I tried "AKSHTTPCustomFeatures=AddonManagerV2Preview,CSIControllersV2Preview", it returned error saying CSIControllersV2Preview doesn't have value associate with the key.

Reply:
I didn't see the place to update custom header, only for managed cluster.

Reply:
The custom headers are usually specified by users, i.e., using this feature at their own risk. If the sub doesn't have `AKSHTTPCustomFeatures` registered, the error message might be strange to the user, since they may have no idea why the feature is required.

Reply:
Yes, there's no common function for this, I'll add one later.

Reply:
The helper functions are implemented in PR [#5342](https://github.com/Azure/azure-cli-extensions/pull/5342).